aws-xray-recorder-sdk-core/src/main/resources/META-INF/native-image/com.amazonaws/aws-xray-recorder-sdk-core/reflect-config.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "name": "com.amazonaws.xray.AWSXRay",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.manifest.SamplingRuleManifest",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.rule.SamplingRule",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.reservoir.Reservoir",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.reservoir.Reservoir$MaxFunction",
```

Comment:
Does it matter that these static nested classes are package-private?

Reply:
No, they have to be added individually.